src/bootstrapper.js

```diff
@@ -0,0 +1,166 @@
+const fs = require('fs')
```

Comment:
I would move this file outside of the `src` directory. I could live at the top of the directory structure. And I would include the generated files in the PR. Because, the generated code is in fact the real package. This file is just a helper file, that facilitates the generation of the files.